examples/basic/android/app/build.gradle

```diff
@@ -83,8 +83,10 @@ android {
     namespace "com.videoplayer"
 
     compileOptions {
-        // These options are necessary to be able to build fro source
-        coreLibraryDesugaringEnabled true
+        // These options are necessary to be able to build from source
```

Comment:
`coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:2.0.4'` in dependencies block is also related and should be commented out

Reply:
As it do not causes any issue, I think it is better to keep it here ...
I want users to do the smallest change possible to enable build from sources.
Is it ok ? @romansavka-mq @KrzysztofMoch ?

Reply:
For me it's okay

Reply:
It strange that doesn't cause issues, I thought that `coreLibraryDesugaring` configuration will not exist if `coreLibraryDesugaringEnabled` is not set to true. But if there is no problem than it's okay